docs/content/manual/_incoming/2436-2524-image-upload-failed-when-http-proxy-configured.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Image upload does not start when HTTP Proxy is configured
+---
+
+* Related issues: [#2436](https://github.com/harvester/harvester/issues/2436) [BUG] Image upload does not start when HTTP Proxy is configured
+* Related issues: [#2524](https://github.com/harvester/harvester/issues/2524) [backport v1.0] [BUG] Image upload does not start when HTTP Proxy is configured
+
+## Category: 
+* Image
+
+## Verification Steps
+1. Clone ipxe-example vagrant project https://github.com/harvester/ipxe-examples
+1. Edit settings.yml
+1. `Set harvester_network_config.offline=true`
```

Comment:
Should be "Set `harvester_network_config.offline=true`"

Reply:
Thanks for the check, correct it accordingly.